crates/polars-ops/src/chunked_array/list/sets.rs

```diff
@@ -99,6 +101,15 @@ where
             set.extend(a);
             out.extend_buf(set.symmetric_difference(set2).copied())
         },
+        SetOperation::IsDisjoint => {
+            // If broadcast `set2` should already be filled.
+            if !broadcast_rhs {
+                set2.clear();
+                set2.extend(b);
```

Comment:
Possible optimization - if len of a and b is known here, then could extend set with smaller one and compare with bigger